e2e-tests/cypress/e2e/journeys/insurance/eligibility/eligible-to-apply-online.spec.js

```diff
@@ -109,12 +109,16 @@ context('Insurance - Eligibility - You are eligible to apply online page - I wan
   });
 
   describe('form submission', () => {
-    it(`should redirect to ${ROUTES.INSURANCE.ELIGIBILITY.ALREADY_HAVE_ACCOUNT}`, () => {
+    it(`should redirect to ${ROUTES.INSURANCE.ROOT}/[referenceNumber]${ROUTES.INSURANCE.ALL_SECTIONS}`, () => {
```

Comment:
same as above - technically, if it was already in main, then this should be here?

Reply:
The other PR was merged into this PR unfortunately. The build issues stemed from this PR as it's the first API integration branch